src/main/java/com/intuit/graphql/orchestrator/stitching/InvalidDirectivePairingException.java

```diff
@@ -0,0 +1,12 @@
+package com.intuit.graphql.orchestrator.stitching;
+
+
+public class InvalidDirectivePairingException extends StitchingException {
+
+  private static final String ERR_MSG = "Field %s in container type %s with resolver directive not allowed "
```

Comment:
might need to update this as well.  Maybe pass the two directives that are not allowed to be together?

Reply:
ok